Dockerfile

```diff
@@ -1,12 +1,23 @@
 FROM composer
 
+FROM php:7.2-cli-stretch
+
+RUN apt-get update && apt-get install git unzip -y
+
 WORKDIR /app
 
-# Install Zenaton
-RUN curl https://install.zenaton.com | sh
+EXPOSE 4001
+
+COPY --from=composer /usr/bin/composer /usr/bin/composer
 
-# Install dependencies
+COPY ./bin /app/bin
+COPY ./src /app/src
+COPY ./start_zenaton /app/start_zenaton
 ADD composer.json composer.lock ./
```

Comment:
I think we can take the opportunity to also change this last `ADD` to a `COPY`. Should work exactly the same.